test/tsconfig.json

```diff
@@ -32,10 +32,10 @@
     "noEmit": true,
     "paths": {
       "multiformats": [
-        "../types/src/index"
+        "../dist/types/src/index"
```

Comment:
I guess this is the reason for the TS errors showing up here, your `gozala/typescript-error-reporter-action` is going to need to get an `npm run build` or `npm run build:ts` in before this happens, maybe just add that to .github/workflows/typecheck.yml?

Reply:
No that change was unintended, should be fixed now.